docs/building.md

```diff
@@ -0,0 +1,158 @@
+# Building the code
+
+This document describes steps needed for compiling the code of the Smart Card
+Connector App and the examples of programmatic communication with this App.
+
+
+## Common building prerequisites
+
+* **OS: Linux**.
+
+  Building under different \*nix system, Mac OS or Windows should be possible
+  too, though most probably will require more efforts.
+
+* The following tools should be present in the system: **bash**, **make**,
+  **curl**, **sed**, **mktemp**, **realpath**, **xxd**.
+
+* **Python 2.7**, including the dev package and the **python-six** module.
+
+  Python 3.x is not supported yet.
+
+* **git** (version 2.2.1+ is recommended).
+
+* **OpenSSL** (version 1.0+ is recommended).
+
+* **Xvfb**.
+
+* (for 64-bit Linux) **32-bit version of libstdc++**.
+
+  For example, on Ubuntu it's provided by the libstdc++6:i386 package.
+
+* **Java Runtime Environment 7**.
+
+In order to **run** the built Apps, you will need *either* of these:
```

Comment:
nit: App should generally not be capitalized according to https://english.stackexchange.com/questions/153696/app-or-app-when-i-refer-to-it-in-a-formal-paper#:~:text=Lowercase%2C%20except%20at%20the%20beginning,should%20consider%20writing%20out%20application. (here and in other places)

Reply:
Done. (I was following the capitalization used throughout Chrome docs, like https://developer.chrome.com/apps/about_apps , but maybe it's more appropriate there than here.)